.gitignore

```diff
@@ -1,4 +1,5 @@
 build/
+Testing/
```

Comment:
Probably not necessary I guess?

Reply:
Correct, I added this when experimenting with the test setup but the committed setup does not need this. I'll remove it in the next commit. :smile: